man/resolved.conf.xml

```diff
@@ -256,6 +256,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerAddress=</varname></term>
+        <listitem><para>Takes a IPv4 address. Specifies the address which the DNS stub resolver will listen for requests
+        on port 53 depending on <varname>DNSStubListener=</varname> UDP, TCP or both. Default to unset.</para>
```

Comment:
`Takes an IPv4 address. Specifies the address on which the DNS stub resolver will listen for requests. When unset, the default of 127.0.0.53:53 is used. When the stub listener is disabled using <varname>DNSStubListener=</varname>, this option has no effect.`

BTW, would it make sense to allow IPv6 too? This would be fairly easy to do. But if there's no particular need, then just IPv4 is OK.

Reply:
Thank you